database/oracle/oracle.Dockerfile

```
@@ -0,0 +1,74 @@
+    net-tools \
+    bc && \
+    rm -rf /var/lib/apt/lists/*
+
+# Create required directories
+RUN mkdir -p /opt/oracle/product/19c/dbhome_1 && \
+    mkdir -p /opt/oracle/oradata && \
+    mkdir -p /opt/oracle/scripts/setup && \
+    mkdir -p /opt/oracle/scripts/startup
+
+# Add Oracle installation files (assume they are in the current directory)
+ADD LINUX.X64_193000_db_home.zip /opt/oracle/
```

Hadolint

[error] Use COPY instead of ADD for files and folders

(DL3020)